<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<testResult type="">
<message column="12" line="21" message="Definition 'a' not used" messageType="warning" number="5000" resource="seqforloopstmt.vdm.result"/>
<message column="28" line="16" message="Name 'SeqForLoopTest`a' is not in scope" messageType="error" number="3182" resource="seqforloopstmt.vdm.result"/>
<message column="8" line="34" message="Expecting sequence type after 'in'" messageType="error" number="3223" resource="seqforloopstmt.vdm.result"/>
<message column="32" line="16" message="Right hand of and is not bool" messageType="error" number="3066" resource="seqforloopstmt.vdm.result"/>
<message column="12" line="15" message="Matching expression is not a product of cardinality 2" messageType="error" number="3205" resource="seqforloopstmt.vdm.result"/>
<message column="15" line="35" message="Name 'SeqForLoopTest`a' is not in scope" messageType="error" number="3182" resource="seqforloopstmt.vdm.result"/>
</testResult>
